doc/man1/openssl-asn1parse.pod

```diff
@@ -26,7 +26,7 @@ B<openssl> B<asn1parse>
 
 =head1 DESCRIPTION
 
-This command is a diagnostic utility that can parse ASN.1 structures.
+This command is a diagnostic aide that can parse ASN.1 structures.
```

Comment:
As a non-native English speaker, I find the word 'aide' not that frequently used, is it better to replace it with 'helper'?

Reply:
Drive by comment: I think "aide" here is not quite correct anyway. An aide is usually a person. I think this should be a "diagnostic aid"....or just use a different word as @InfoHunter suggests :-)

Reply:
Haha, this my first time to encounter the word 'aide', very impressive anyway.

Reply:
Looking at https://www.thesaurus.com/browse/tool and other references, I couldn't find anything better.  And Matt is of course right, "aide" should be "aid"  Argh.  Fixup commit pushed.  Helper doesn't quite convey the right meaning to me.  If folks want it changed -- or they want me to remove the find-doc-nits check (see ```man1words``` in that script), let me know.

Reply:
What's wrong with using tool at this place? The word aid looks awkward here to me (although of course I am not native English speaker).
I would remove the check for the tool word.

Reply:
yes, reverted.